pkg/jobqueue/jobqueue.go

```diff
@@ -92,6 +92,9 @@ type JobQueue interface {
 
 	// Deletes the worker
 	DeleteWorker(workerID uuid.UUID) error
+
+	// AllRootJobIDs returns a list of top level job UUIDs that the worker knows about
+	AllRootJobIDs() ([]uuid.UUID, error)
```

Comment:
I don't think that belongs at this layer. If the caller wants to filter by channel that is a separate issue. The goal here is to list everything so that composer-cli can let the user examine what's happening.